### ISSUE TITLE
Requirements.txt version mistake?

### ISSUE THREAD

**Issue by markdregan** [AutoViML/featurewiz#55]
Should the requirements for some libraries be >= as opposed to ~=

featurewiz 0.1.991 requires Pillow~=9.0.0, but you have pillow 9.2.0 which is incompatible.
featurewiz 0.1.991 requires scikit-learn~=0.24, but you have scikit-learn 1.1.2 which is incompatible.

For scikit-learn, 0.24 is very old. And you're preventing more recent versions of scikit-learn from being usable. 

Am I misunderstanding?

**Comment by AutoViML** [comment]
Hi @markdregan : 
Thanks for trying out featurewiz 👍 

We created the requirements.txt file after looking at various platforms like Kaggle, Colab, etc and tried to find the best fit of versions for them to run featurewiz. That's why those versions are listed like that. If you want to install featurewiz smoothly in your environment, just do the following:
```
 conda install -c conda-forge featurewiz
```

If that is not possible (since conda is not available in Kaggle Kernels), you must try this:
```
 !pip install featurewiz --ignore-installed --no-deps
 !pip install xlrd --ignore-installed --no-deps 
 !pip install Pillow==9.0.0
```

I have found that these three steps work in Kaggle, Colab etc. Hope you succeed in installing them.
AutoVimal
